globus.md

```diff
@@ -0,0 +1,88 @@
+---
+highlighter: none
+layout: default
+title: Using Globus to Transfer Files to and from CHTC
+---
+
+[Globus](https://www.globus.org/) is a data management service that lets you
+move files files between **endpoints**, computers that are connected to the
+Globus file transfer network.
+CHTC maintains a Globus endpoint that can serve as either the
+source or destination of a Globus file transfer
+(i.e., you can move your files out of CHTC to some other Globus endpoint,
+or from some other Globus endpoint to CHTC).
+
```

Comment:
This is where I would usually include the doc thesis: 
"this guide will show you how to transfer files to and from CHTC using our Globus endpoint." 

And then maybe put the following paragraph under its own heading, like "Requirements" or "Prerequisites"

Reply:
Also - somewhere you should talk about when/why you would use Globus (big file transfers, coming from a Globus endpoint, etc.). Maybe in the intro here?

Reply:
I made a Prerequisites section, and reworked the introduction to note where Globus is most useful (primary when someone else wants to you to use Globus, for whatever reason).